mxp/mxpOnOff.cpp

```
@@ -14,35 +14,307 @@
+      pMarker->pAction = pReplacementAction;
+      }
+
+    pDoc->m_ActiveTagList.RemoveAt (current);
+    delete pTag;
+    }
+  }
+
+static void DiscardMXPTagsCreatedAfter (CMUSHclientDoc * pDoc,
+                                        const __int64 iBoundary)
+  {
+  for (POSITION pos = pDoc->m_ActiveTagList.GetHeadPosition (); pos; )
```

Clang (14.0.6)

[warning] variable 'pos' is not initialized

(cppcoreguidelines-init-variables)